exporter/otlphttpexporter/otlp.go

```diff
@@ -58,7 +58,7 @@ const (
 )
 
 // Crete new exporter.
-func newExporter(cfg config.Exporter, logger *zap.Logger, buildInfo component.BuildInfo) (*exporter, error) {
+func newExporter(cfg config.Exporter, logger *zap.Logger, settings component.TelemetrySettings, buildInfo component.BuildInfo) (*exporter, error) {
```

Comment:
I think is better to accept the entire `component.ExporterCreateSettings` since contains temeletrysettings and build info.